### PR TITLE
Olhaholiak/categories renaming

### PR DESCRIPTION
**Pull request by OlhaHoliak**
To implement a new migration the database needs to be deleted first.

### CODE REVIEW

**Comment by OlhaHoliak** [comment]
Migration was changed not to drop tables, so we don't need to delete the database.

**Review by v-ivanchuk — APPROVED** [review]

File: OutOfSchool/OutOfSchool.DataAccess/Repository/IDepartmentRepository.cs [hunk 1]
```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
```
Comment (yfedo) [inline OutOfSchool/OutOfSchool.DataAccess/Repository/IDepartmentRepository.cs:3]: dead usings

File: OutOfSchool/OutOfSchool.DataAccess/Repository/IDepartmentRepository.cs [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+{
+    public interface IDepartmentRepository : IEntityRepository<Department>
+    {
+        /// <summary>
+        /// Checks entity elements for uniqueness.
+        /// </summary>
+        /// <param name="entity">Entity.</param>
+        /// <returns>Bool.</returns>
+        bool SameExists(Department entity);
+
```
Comment (yfedo) [inline OutOfSchool/OutOfSchool.DataAccess/Repository/IDepartmentRepository.cs:16]: this method is used in 2 repositories, maybe can be used to base interface or separate interface...

File: OutOfSchool/OutOfSchool.DataAccess/Repository/IClassRepository.cs [hunk 3]
```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using OutOfSchool.Services.Models;
+
```
Comment (yfedo) [inline OutOfSchool/OutOfSchool.DataAccess/Repository/IClassRepository.cs:5]: dead usings

File: OutOfSchool/OutOfSchool.DataAccess/Repository/ClassRepository.cs [hunk 4]
```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using OutOfSchool.Services.Models;
+
```
Comment (yfedo) [inline OutOfSchool/OutOfSchool.DataAccess/Repository/ClassRepository.cs:6]: dead usings

File: OutOfSchool/OutOfSchool.DataAccess/Repository/DepartmentRepository.cs [hunk 5]
```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using OutOfSchool.Services.Models;
```
Comment (yfedo) [inline OutOfSchool/OutOfSchool.DataAccess/Repository/DepartmentRepository.cs:5]: also dead usings))

**Review by yfedo — APPROVED** [review]

File: OutOfSchool/OutOfSchool.DataAccess/Repository/IDepartmentRepository.cs (hunk 1 above)
Comment (OlhaHoliak) [reply OutOfSchool/OutOfSchool.DataAccess/Repository/IDepartmentRepository.cs:3]: thank you, deleted :)

File: OutOfSchool/OutOfSchool.DataAccess/Repository/IDepartmentRepository.cs (hunk 2 above)
Comment (OlhaHoliak) [reply OutOfSchool/OutOfSchool.DataAccess/Repository/IDepartmentRepository.cs]: I separated the interface.

**Review by DmyMi — APPROVED** [review]

**Pull request merged**